src/store.js

```diff
@@ -355,11 +357,25 @@ export class Store {
     };
   }
 
-  // Todo: find a better name for this function.
   refresh() {
+    if (this._cacheEnabled === false) {
```

Comment:
No,

if the cache is disabled, then we need to clear it.
otherwise we do nothing because it is enabled by default in the client.

anyway, it needs tests